TODO.md

```diff
@@ -0,0 +1,40 @@
+This serves as a braindump for features that we want to scope for V1 (or seed a V2, V3, etc)
+
+# Project
+
+## Runtime
+
+- Node.js version? Do we have a minimum?
+  - @Nick: I would be fine with 14.x+ to jettison some legacy stuff but do not know if that would wall us off
+
+## Publication
+
+- Is there an NPM namespace / package we want to publish under? `@orkes/conductor-js` or `@conductor/client`
+  - it looks like conductor is taken but has nothing so we may be able to grab it https://www.npmjs.com/org/conductor?activeTab=packages
+- CI/CD rails
+  - Github actions
+  - Auto publication
+  - Use something like [semantic release?](https://github.com/semantic-release/semantic-release)
+
+## Formatting / linting
+
+- .editorconfig
+- prettier?
+  - helps reduce editor churn and space/tab bikeshedding
+  - [maybe use husky to automate](https://www.npmjs.com/package/husky)
+
+## Tests
+
+- Unit tests
+  - Framework: jest? Mocha?
+- Integration tests
+  - It'd be _awesome_ if we could use an actual conductor cluster running in docker for integration style tests
```

Comment:
Huge plus one to this. I think we can easily provision and maintain this - similar to play.orkes.io.